layers/drawdispatch/descriptor_validator.cpp

```diff
@@ -338,8 +340,9 @@ bool DescriptorValidator::ValidateDescriptor(const DescriptorBindingInfo &bindin
     const auto *variable = FindMatchingImageVar(binding_info.second, image_view_ci);
```

Comment:
```suggestion
    const auto *image_variable = FindMatchingImageVar(binding_info.second, image_view_ci);
```
or maybe better:
```suggestion
    const auto *image_spirv_var = FindMatchingImageVar(binding_info.second, image_view_ci);
```
since variable is a bloated term